cmd/vaultworker/res/configuration.toml

```diff
@@ -24,6 +24,12 @@ scheme = "https"
 server = "127.0.0.1"
 port = "8200"
 certpath = "v1/secret/edgex/pki/tls/edgex-kong"
+coredatasecretpath = "v1/secret/edgex/coredata/mongodb"
```

Comment:
the configuration.toml file is not being used currently. we will make it consistent with configuration-docker.toml file for now and update both files in the future, which is being tracked under #75 .

Reply:
Again this is something I can't rightfully approve of until it's resolved in this PR specifically, because merging half finished work is not something I approve of